.vscode/launch.json

```diff
@@ -10,10 +10,11 @@
             "name": "Launch CLI - up",
             "program": "${workspaceFolder}/src/spec-node/devContainersSpecCLI.ts",
             "cwd": "${workspaceFolder}",
+            // "envFile": "${workspaceFolder}/.env",  // Create .env file in the workspace folder to pass environment variables to the CLI.
             "args": [
                 "up",
                 "--workspace-folder",
-                "../devcontainers-features",
+                "/home/node/example-project",
```

Comment:
Is this path guaranteed to exist? What's the reason for moving away from launching against the `../devcontainers-features`

Reply:
The previous path also doesn't exist by default.  I think it's worth checking in a scaffold project (and maybe .gitignore it so people can edit/debug configs from the barebones template).